docs/pages/javascript.md

```diff
@@ -79,8 +79,8 @@ Sadly, the "static analysis" promised by these bundlers to detect unused code in
 ```js
 // /!\ Don't use this if tree shaking works with standard named imports.
 // Only Dropdown and DropdownMenu will be included in your application.
-import Dropdown from 'foundation-sites/js/foundation.dropdown';
-import DropdownMenu from 'foundation-sites/js/foundation.dropdownMenu';
+import { Dropdown } from 'foundation-sites/js/foundation.dropdown';
+import { DropdownMenu } from 'foundation-sites/js/foundation.dropdownMenu';
```

Comment:
Looks Good to me!

Reply:
Confimed with the export code too
1. https://github.com/zurb/foundation-sites/blob/develop/js/foundation.dropdown.js#L431
2. https://github.com/zurb/foundation-sites/blob/develop/js/foundation.dropdownMenu.js#L483

Merging it...